src/handle-command.ts

```diff
@@ -34,15 +34,9 @@ export async function handleCommand({
     const answer = await askQuestion(rl, `Run this command? (y/n) `);
     if (answer === "y") {
       console.log(chalk.green("Executing command: "), chalk.blue(fullCommand));
-      const proc = exec(fullCommand, async (error, stdout, stderr) => {
-        if (error) {
-          await sendChat({ isError: true, message: error.message, rl });
```

Comment:
Oo, just realized this removes the sendChat on stderr.

I want to automatically sendChat on stderr so that the AI will return a new recommended command based on the previous command's error message.